hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/states/UIErrorState.kt

```diff
@@ -59,6 +59,6 @@ private fun OrchestrationMessage.UIException.toDescription(): UIErrorDescription
         title = "UI Exception",
         message = message,
         stacktrace = stacktrace,
-        recovery = { OrchestrationMessage.RetryFailedCompositionRequest().sendAsync() }
+        recovery = { OrchestrationMessage.RecompileRequest().sendAsync() }
```

Comment:
This change is not necessarily clear to me:
It seems like you want to recover from the UIException by issueing a full reload?
However, this is effectively a noop in continuous builds and if nothing has changed, IIRC nothing will be done as well?

Reply:
Agree, this change does not make a lot of sense, will revert it